app/src/views/home/maximize_screen.tsx

```diff
@@ -83,6 +83,22 @@ export default function MaximizeScreen({
           </ThemedView>
         </ThemedView>
 
```

Comment:
This is good thank you, good idea to mask the location button if there isn't any